common/app/views/fragments/containers/facia_cards/standardContainer.scala.html

```diff
@@ -14,7 +14,7 @@
         ("js-container--fc-show-more", containerDefinition.addShowMoreClasses),
         ("fc-show-more--mobile-only", containerDefinition.hasMobileOnlyShowMore)
     ))"
-         data-title="@Html(Localisation(containerDefinition.displayName getOrElse ""))"
+         data-title="@Localisation(containerDefinition.displayName getOrElse "")"
```

Comment:
@sndrs seems to have escaped it fine
![image](https://cloud.githubusercontent.com/assets/7304387/8088830/10635eaa-0f9c-11e5-9459-49a92019788d.png)